src/xtgeo/well/well1.py

```diff
@@ -371,10 +372,20 @@ def zonelogname(self, zname):
     @property
     def dataframe(self):
         """Returns or set the Pandas dataframe object for all logs."""
+        warnings.warn(
+            "Direct access to the dataframe property will be deprecated in xtgeo 5.0. "
+            "Use `get_dataframe()` instead.",
+            PendingDeprecationWarning,
+        )
```

Comment:
I think this OK. When we come to version 4, we can change it to DeprecationWarning unless we change our mind.